superset-frontend/src/features/home/Menu.tsx

```diff
@@ -64,6 +64,7 @@ const StyledHeader = styled.header`
           ${theme.gridUnit}px
           ${theme.gridUnit * 4}px;
         max-width: ${theme.gridUnit * theme.brandIconMaxWidth}px;
+        margin-top: 2px;
```

Comment:
Please use `theme` variable here as in previous rules

Reply:
This change can be considered as an upstream pull request (up to 3 hours of internal time allowed to open PR to upstream)

Reply:
Changed. Thanks!